assets/grafana/celeborn-dashboard.json

```diff
@@ -334,7 +334,7 @@
                 "type": "prometheus",
                 "uid": "${DS_PROMETHEUS}"
               },
-              "expr": "metrics_ApplicationCount_Value",
+              "expr": "metrics_ApplicationCount_Value{role=\"Master\"}",
```

Comment:
We need to add the worker's application count panel.

Reply:
@FMX, I have reverted this change. Meanwhile, this panel could provide the application count of master and worker instances.